README.md

```diff
@@ -11,15 +11,16 @@
 
 ## fast.ai 코스 관련
 
-| 완료 | 진행중 | 원제목 | 원링크 | 번역제목 | 번역링크 | 
+| Status | 원제목 | 원링크 | 번역제목 | 번역링크 | 
 | -- | -- | ------- | ---- | ------- | ---- |
-| <ul><li>- [ ] </li> | <ul><li>- [X] </li> | Understanding Fastai's fit_one_cycle method | [Link](https://iconof.com/1cycle-learning-rate-policy/) | 미정 | [링크](https://github.com/jehyunlee/texts/blob/master/1cycle-learning-rate-policy/text.md)
+| complete  | [Link]() | 미정 | [링크]()
 
 ## 머신러닝/딥러닝 일반
 
 | 완료 | 진행중 | 원제목 | 원링크 | 번역제목 | 번역링크 | 
 | -- | -- | ------- | ---- | ------- | ---- |
-| <ul><li>- [ ] </li> | <ul><li>- [ ] </li> | Estimating an Optimal Learning Rate For a Deep Neural Network  | [Link](https://towardsdatascience.com/estimating-optimal-learning-rate-for-a-deep-neural-network-ce32f2556ce0) | 미정 | [링크]()
+| Complete | Understanding Fastai's fit_one_cycle method  | [Link](https://iconof.com/1cycle-learning-rate-policy/) | Fast.ai의 fit_one_cycle method 이해하기 | [링크](https://github.com/jehyunlee/texts/blob/master/1cycle-learning-rate-policy/text.md)|
+| Complete | Estimating an Optimal Learning Rate For a Deep Neural Network  | [Link](https://towardsdatascience.com/estimating-optimal-learning-rate-for-a-deep-neural-network-ce32f2556ce0) | 심층신경망에 맞는 최적 학습률 평가 | [링크](https://github.com/jehyunlee/texts/blob/master/estimating_an_optimal_learning_rate_for_a_deep_neural_network/text.md)
```

Comment:
Can you write in Korean instead of _Complete_ ?

Reply:
I did it!